### PR TITLE
Add wall test

### PR DESCRIPTION
**Pull request by zora-wang**
PR to Workflow Tests with Revit nodes.
@monikaprabhu ,
@riteshchandawar

### CODE REVIEW

**Comment by zora-wang** [comment]
@monikaprabhu  @riteshchandawar , please help to review it

File: test/Libraries/RevitIntegrationTests/WorkflowTests.cs [hunk 1]
```diff
@@ -0,0 +1,231 @@
...
+using System.Collections.Generic;
+using RevitServices.Transactions;
+
+namespace RevitSystemTests
+{
+    [TestFixture]
+    public class WorkflowTests : RevitSystemTestBase
+    {
+        //this method will be deleted on Monday after Mr.Ritesh add it to RevitSystemTestBase
+        protected static IList<Autodesk.Revit.DB.Wall> GetAllWalls()
```
Comment (riteshchandawar) [inline test/Libraries/RevitIntegrationTests/WorkflowTests.cs]: Zora, As discussed, we need to move this method to RevitSystemTestBase class. I have added here to show you how can you access Walls from Revit.

File: test/Libraries/RevitIntegrationTests/WorkflowTests.cs [hunk 2]
```diff
@@ -0,0 +1,231 @@
...
+            Assert.AreEqual(33, model.CurrentWorkspace.Connectors.Count());
+
+            // Check StructuralFraming.BeamByCurve
+            var beam = "bcef7f1f-fff5-48bc-8e78-7302e7344be8";
+            AssertPreviewCount(beam, 23);
+
+            // get all StructuralFraming.BeamByCurve.
+            for (int i = 0; i <23; i++)
+            {
+                var allbeam = GetPreviewValueAtIndex(beam, i);
```
Comment (riteshchandawar) [inline test/Libraries/RevitIntegrationTests/WorkflowTests.cs]: this one will not check whether that is actually creating Beam or not, update this to 
var allbeam = GetPreviewValueAtIndex(beam, i) as StucturalFraming;

**Comment by zora-wang** [comment]
Closing this PR as there are some missing files and redoing all will take time so creating new PR from another branch addWorkflowTests, will send PR from there.

-Zora

**Pull request closed without merging**